virtual-desktop/src/app/start-url-manager/start-url-manager.service.ts

```diff
@@ -68,7 +72,11 @@ export class StartURLManager implements MVDHosting.LoginActionInterface {
     } catch (e) {
       // this.logger.warn('Context Data(%s) specified in app2app query URL parameter is not valid JSON', args.contextData);
       this.logger.warn('ZWED5196W', args.contextData);
-      return false;
+      if (args.actionType == 'launch' && args.formatter == 'data') { // If we know what the plugin is, we can still try to launch it w/o app2app data
```

Comment:
I am wondering why you check only for `args.actionType == 'launch'`? What about `actionType` of `message`?

Reply:
This is so that apps still spawn with bad app2app data. It's specifically for launch